src/jats/importer/jats-body-transformations.ts

```diff
@@ -401,6 +406,83 @@ export const jatsBodyTransformations = {
       }
     }
   },
+  moveAffiliationsToBody(
+    doc: Document,
+    affiliations: Build<Affiliation>[] | undefined,
+    body: Element,
+    createElement: (tagName: string) => HTMLElement
+  ) {
+    if (affiliations?.length) {
+      const section = createElement('sec')
+      section.setAttribute('sec-type', 'affiliations')
+      affiliations.forEach((affiliation) => {
+        const item = doc.createElement('aff')
+        item.setAttribute('id', affiliation._id)
+        affiliation.institution &&
+          item.setAttribute('institution', affiliation.institution)
+        affiliation.email &&
+          item.setAttribute('email', JSON.stringify(affiliation.email))
+        affiliation.department &&
+          item.setAttribute('department', affiliation.department)
+        affiliation.addressLine1 &&
+          item.setAttribute('addressLine1', affiliation.addressLine1)
+        affiliation.addressLine2 &&
+          item.setAttribute('addressLine2', affiliation.addressLine2)
+        affiliation.addressLine3 &&
+          item.setAttribute('addressLine3', affiliation.addressLine3)
+        affiliation.postCode &&
+          item.setAttribute('postCode', affiliation.postCode)
+        affiliation.country && item.setAttribute('country', affiliation.country)
+        ;(affiliation.priority == 0 || affiliation.priority) &&
```

Comment:
@mnatsheh FYI after your review, I added priority to each affiliations & contributors model, as manuscripts-api failed to validate these models.
(I also had to add them to the encoder/decoder/jats-body-parser)